src/Coriolis/non_traditional_beta_plane.jl

```diff
@@ -0,0 +1,80 @@
+"""
+    NonTraditionalBetaPlane{FT} <: AbstractRotation
+
+A Coriolis implementation that accounts for the latitudinal variatino of both
+the locally vertical and the locally horizontal components of the rotation vector.
+Traditionally (see [`BetaPlane`](@ref)) only the locally vertical component is
```

Comment:
I guess if we are going to use the word "Tradiationally" we probably should supply at least a little context (that we are talking about ocean modeling, rather than some other field). Perhaps we can say:

> The "traditional" approximation in ocean models accounts for only the locally vertical component of the Coriolis vector.